scripts/test

```diff
@@ -11,18 +11,20 @@ if [ -n "$res" ]; then
   exit 1
 fi
 
-git diff --exit-code 
-if [ "$?" -gt "0" ]; then
-  exit 1
-fi
+# where necessary attempts to add/remove modules from go.mod and go.sum.
+echo "running go mod tidy ..."
+go mod tidy
 
```

Comment:
Let's remove this blank line so it's clear the `git diff --exit-code` is running in response to the `go mod tidy` command.